wiki/History_of_osu!/Online_rankings/osu!catch/en.md

```diff
@@ -20,10 +20,18 @@ Over the years, there were three different systems used for ranking players:
 
 Because of this, this list only includes players who have reached rank #1 on the main leaderboard that was used at the time. *Italicised* dates and numbers indicate that the corresponding player is continuing their reign as the osu!'s rank #1 holder either from the previous year or into the following year.
 
+## 2018
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ::{ flag=RU }:: [skvix](https://osu.ppy.sh/users/4447639) | *2018-04-14* | *2019-04-06* | [\[1\]](https://web.archive.org/web/20180414205044/https://osu.ppy.sh/rankings/fruits/performance) |
```

Comment:
while the rankings page is a fine source for showing that they were rank 1 at the time of the snapshot, it doesn't show that the "From" and "To" dates are accurate